src/main/pages/che-7/installation-guide/proc_deploy-che-with-self-signed-tls-on-kubernetes.adoc

```diff
@@ -0,0 +1,50 @@
+[id="deploy-{prod-id-short}-with-self-signed-tls-on-kubernetes_{context}"]
+= Deploy {prod-short} with self signed TLS certificates on Kubernetes
+
+This section describes how to deploy {prod-short} with self-signed TLS certificates on Kubernetes platform.
+
+
+[discrete]
+== Prerequisites
+
+* Have all required keys/certificates generated
+
+
+[discrete]
+== Procedure
+
+. Pre-create namespace for {prod-short}
+
++
+[subs="+quotes"]
+----
+$ kubectl create namespace che
+----
+
+
+. Create secret with domain key and certificate
+
++
+[subs="+quotes"]
+----
+$ kubectl create secret tls che-tls --key=domain.key --cert=domain.crt -n che
+----
+
+
+. Create secret from CA certificate
+
++
+[subs="+quotes"]
+----
+$ cp rootCA.crt ca.crt
```

Comment:
why do we need this `cp`?

Reply:
Not sure, but i think to avoid actions  directlty on root cert and reduce possibility to broke it.

Reply:
I guess because we "hardcode" it everywhere https://github.com/eclipse/che/blob/master/deploy/kubernetes/helm/che/templates/deployment.yaml#L102